src/components/Select/Select.scss

```diff
@@ -159,11 +163,6 @@ $stacking-order: (
     border-radius: var(--p-border-radius-base);
     background-color: var(--p-surface);
     @include focus-ring($border-width: rem(1px));
-
-    // 'position' needs to sit below focus-ring since it will be overwritten
```

Comment:
This change should be reverted, `position` needs to be in this order.

Reply:
Unfortunately the CLI formatter keeps making this change and not respecting the disabled lint